service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerSubmittedTest.java

```diff
@@ -85,17 +86,16 @@ void shouldBuildNotificationTemplatesWithCompleteValues() throws Exception {
         verify(notificationClient).sendEmail(
             eq(CAFCASS_SUBMISSION_TEMPLATE), eq("cafcass@cafcass.com"), eq(expectedCafcassParameters), eq("12345")
         );
+
+        verify(notificationClient, never()).sendEmail(
+            eq(HMCTS_COURT_SUBMISSION_TEMPLATE), eq("FamilyPublicLaw+ctsc@gmail.com"),
+            eq(expectedHmctsParameters), eq("12345")
+        );
     }
 
     @Test
     void shouldBuildNotificationTemplatesWithValuesMissingInCallback() throws Exception {
-        CallbackRequest request = CallbackRequest.builder()
-            .caseDetails(CaseDetails.builder()
-                .id(12345L)
-                .data(ImmutableMap.<String, Object>builder()
-                    .put("caseLocalAuthority", "example").build())
-                .build())
-            .build();
+        CaseDetails caseDetails = buildCaseDetails(false);
```

Comment:
this is not good. What does true false mean with buildCaseDetails, it is not clear.